evaluator/images/clang-tidy/build_urls.py

```diff
@@ -29,19 +32,64 @@ def get(url):
 def process(check):
```

Comment:
We're trying to put type hints wherever we touch old codebase. Could you do it for `check` parameter, please?